doc/content/getting-started/cli/create-application/_index.md

```diff
@@ -38,3 +38,43 @@ $ ttn-lw-cli applications link set app1 --api-key NNSXS.VEEBURF3KR77ZR..
 ```
 
 Your application is now linked. You can now use the builtin MQTT server and webhooks to receive uplink traffic and send downlink traffic.
+
+## Set message payload formatter
+
+> For more information on message payload formatters, see [Payload Formatters]({{< ref "/integrations/payload-formatters" >}})
+
+Default message payload formatters for an application can be set when linking with a Network Server.
+
+Write the downlink and uplink payload formatters in `down.js` and `up.js` respectively:
+
+```js
+/* down.js */
+function Encoder(payload, f_port) {
+  if (payload.enabled === true) {
+    return [1];
+  }
+  return [0];
+}
+```
+
+```js
+/* up.js */
+function Decoder(bytes, f_port) {
+  return {
+    raw: bytes
+  }
+}
+```
+
+Then, link the application, including the message payload formatter flags:
+
+```bash
+$ ttn-lw-cli applications link set app1 \
+  --api-key NNSXS.VEEBURF3KR77ZR..
+  --default-formatters.down-formatter FORMATTER_JAVASCRIPT \
```

Comment:
does this also work for `FORMATTER_CAYENNELPP` and `FORMATTER_DEVICEREPO`

Reply:
Yes, and for these two the `...-formatter-parameter` is not needed.